script.js

```diff
@@ -0,0 +1,43 @@
+window.addEventListener('load', e => {
+
+    const monthlyPriceBasic = 19.99;
+    const monthlyPriceProfessional = 24.99;
+    const monthlyPriceMaster = 39.99;
+
+    const yearlyPriceBasic = 199.99;
+    const yearlyPriceProfessional = 249.99;
+    const yearlyPriceMaster = 399.99;
```

Comment:
Teniendo en mente que estas 6 constantes se refieren a un mismo tema: el price, y que son secuenciales, se podria estructurar un objeto (lo que en python era un diccionario)

ademas, consultando una API, te terminaria respondiendo un JSON que seguramente seguiria esta estructura que te propongo:


```suggestion
  	const price = {
		monthly: {
			basic: 19.99,
			professional: 24.99,
			master: 39.99
		},
		yearly: {
			basic: 199.99,
			professional: 249.99,
			master: 399.99
		}
	}
```

que opinas?